src/network/networkd-routing-policy-rule.c

```diff
@@ -836,6 +862,109 @@ int config_parse_routing_policy_rule_device(
         return 0;
 }
 
+int config_parse_routing_policy_rule_port_range(const char *unit,
+                                                const char *filename,
+                                                unsigned line,
+                                                const char *section,
+                                                unsigned section_line,
+                                                const char *lvalue,
+                                                int ltype,
+                                                const char *rvalue,
+                                                void *data,
+                                                void *userdata) {
```

Comment:
Please use the same indentation for `config_parse_routing_policy_rule_protocol()`.